[test_scheduling_features] AssertionError: Wrong platforms ([]) in build-extensions-browser

Task "bugbug train test label selection model" is failing (https://community-tc.services.mozilla.com/tasks/OoW7LdsjSm6Ip089ceiN2g/runs/1/logs/public/logs/live.log#L388):


```
Traceback (most recent call last):
  File "/usr/local/bin/bugbug-train", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.12/site-packages/scripts/trainer.py", line 143, in main
    retriever.go(args)
  File "/usr/local/lib/python3.12/site-packages/scripts/trainer.py", line 41, in go
    metrics = model_obj.train(limit=args.limit)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/bugbug/model.py", line 373, in train
    X = self.extraction_pipeline.transform(X_gen)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sklearn/pipeline.py", line 1092, in transform
    Xt = transform.transform(Xt, **routed_params[name].transform)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sklearn/utils/_set_output.py", line 319, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/bugbug/commit_features.py", line 999, in transform
    res = feature_extractor(commit["test_job"], commit=commit)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/bugbug/test_scheduling_features.py", line 30, in __call__
    assert len(platforms) == 1, "Wrong platforms ({}) in {}".format(
           ^^^^^^^^^^^^^^^^^^^
AssertionError: Wrong platforms ([]) in build-extensions-browser
```

--- a/bugbug/test_scheduling.py
+++ b/bugbug/test_scheduling.py
@@ -114,6 +114,7 @@ JOBS_TO_IGNORE = (
     "jittest",
     "jsreftest",
     "android-hw-gfx",
+    "build-extensions-browser",
 )
 
 

